lib/peeps.rb

```diff
@@ -0,0 +1,33 @@
+require 'pg'
+
+class Peeps
+
+  attr_reader :id, :message
+
+  def initialize(id:, message:)
+    @id = id
+    @message = message
+  end
+
+  def self.all
+    if ENV['ENVIRONMENT'] = 'test'
+      connection = PG.connect(dbname: 'peep_manager_test')
+    else
+      connection = PG.connect(dbname: 'peeps_manager')
+    end
+
+    result = connection.exec("SELECT * FROM peeps;")
+    result.map { |peep| peep['message'] }
+  end
+
+  def self.create(newpeep:)
+    if ENV['ENVIRONMENT'] == 'test'
+      connection = PG.connect(dbname: 'peep_manager_test')
+    else
+      connection = PG.connect(dbname: 'peeps_manager')
+    end
+
+    connection.exec("INSERT INTO peeps (message) VALUES('#{newpeep}')")
```

Comment:
clear and good use of the SQL syntax, as well as a clear understanding of how to integrate it into ruby